Job withdrawals, member/group renames (#5)
SRCF/srcf-python: srcf/controllib/emails/member/name.txt

> The preferred name we have on record for you has now been changed from {{ old_name }} to {{ new_name }}.

Reviewer: Do we even need the template variables?  The new name will be in the "Hello foo," at the top.  I think a static message to confirm that the name has been updated will suffice (and avoid potential deadnaming).

Author: ```suggestion
This is just to confirm that your preferred name has now been updated in our records.
```

And a similar line for group accounts, as the current one there is also a bit redundant:

```diff
-The description for the {{ target.society }} group account has now been changed from "{{ old_description }}" to "{{ new_description }}".
+This is just to confirm that the description of the {{ target.society }} group account (previously "{{ old_description }}") has now been updated in our records.
```

---

However, it occurs to me that the email is addressed to the job's `self.owner` / `self.society`, which probably means the old name.  I think we want to reassign that field during the execution of name (and email?) jobs, or at least obtain the updated `Member` / `Society` objects from SRCFLib and use them for the emails until SRCFLib can handle email itself.
